### PR TITLE
Remove "Attend KubeCon in Shanghai" button

### PR DESCRIPTION
**Pull request by ScottBrenner**
Been a month since the event, safe to remove the "Attend" button?

### CODE REVIEW

File: content/en/_index.html [hunk 1]
```diff
@@ -44,10 +44,6 @@ <h2>The Challenges of Migrating 150+ Microservices to Kubernetes</h2>
         <br>
         <br>
         <br>
-        <a href="https://www.lfasiallc.com/events/kubecon-cloudnativecon-china-2018/" button id="desktopKCButton">Attend KubeCon in Shanghai on Nov. 13-15, 2018</a>
-        <br>
-        <br>
-        <br>
         <br>
         <a href="https://events.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2018/" button id="desktopKCButton">Attend KubeCon in Seattle on Dec. 11-13, 2018</a>
```
Comment (xiangpengzhao) [inline content/en/_index.html]: Since you're here, could you please change the date `Dec. 11-13` to `Dec. 10-13` as the related event page shows?

Comment (xiangpengzhao) [reply content/en/_index.html]: Guess we want to merge this PR before Dec. 13, otherwise it will be outdated again :)

Comment (ScottBrenner) [reply content/en/_index.html]: Just removed a few more Shanghai mentions and changed the Seattle dates.

**Comment by xiangpengzhao** [comment]
@ScottBrenner thanks for the contribution! Will let the bot merge this PR in case we miss the train to Seattle :)
/lgtm
/approve

**Comment by kbarnard10** [comment]
I would prefer to merge this fix: https://github.com/kubernetes/website/pull/11136

**Pull request closed without merging**